config/locales/client.en.yml

```diff
@@ -873,11 +873,17 @@ en:
           prefill:
             title: "Prefill with settings for:"
             gmail: "Gmail"
+            outlook: "Outlook"
```

Comment:
I think the official name of the hosted service is "Outlook.com", Outlook is the name of all their email clients.

Reply:
Ah yes I had it outlook.com first then changed it to this because I wasn't sure, I think you are right. Will change back